AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/MutationSync/OutgoingMutationQueue/OutgoingMutationQueue.swift

```diff
@@ -138,6 +138,8 @@ final class OutgoingMutationQueue: OutgoingMutationQueueBehavior {
                        mutationEventPublisher: MutationEventPublisher) {
         log.verbose(#function)
         self.api = api
+
+        dispatchOutboxStatusEvent(isEmpty: checkStatus())
```

Comment:
`checkStatus` was suggestion for the new responder method, if you are using the existing start method, then i would expect `checkStatus` to do one thing primary, which is to check the "status" which is "the number of mutation events to be processed" and then secondarily, you can dispatch the event as part of the check... but the function would still return void

if you want it to return a bool, name it like `isMutationEventsEmpty()`, then you can use it to dispatch the outboxStatusEvent